test/NisCodeService.Sync.OrganizationRegistry.Tests/DynamoDbNisCodeStorageTests.cs

```diff
@@ -22,13 +23,17 @@ public async Task DoesPersistKvps()
         {
             using var _ = DockerComposer.Compose("dynamodb.yml", "niscode-sync-integration-dynamo-test");
```

Comment:
We're also using the same docker-compose file here as in the fixture. This will cause issues I think.

Reply:
It's a different file, with different port mappings.